packages/kbn-esql-validation-autocomplete/src/validation/validation.test.ts

```diff
@@ -5077,6 +5077,33 @@ describe('validation logic', () => {
         );
         testErrorsAndWarnings('from a_index | eval mv_zip(null, null, null)', []);
         testErrorsAndWarnings('row nullVar = null | eval mv_zip(nullVar, nullVar, nullVar)', []);
+        testErrorsAndWarnings('row var = mv_zip(to_string(true), to_string(true))', []);
+        testErrorsAndWarnings(
+          'from a_index | where length(mv_zip(stringField, stringField)) > 0',
+          []
+        );
+
+        testErrorsAndWarnings(
+          'from a_index | where length(mv_zip(booleanField, booleanField)) > 0',
+          [
+            'Argument of [mv_zip] must be [string], found value [booleanField] type [boolean]',
+            'Argument of [mv_zip] must be [string], found value [booleanField] type [boolean]',
+          ]
+        );
+
+        testErrorsAndWarnings('from a_index | eval var = mv_zip(stringField, stringField)', []);
+
+        testErrorsAndWarnings(
+          'from a_index | eval var = mv_zip(to_string(booleanField), to_string(booleanField))',
+          []
+        );
+
+        testErrorsAndWarnings('from a_index | eval mv_zip(booleanField, booleanField)', [
+          'Argument of [mv_zip] must be [string], found value [booleanField] type [boolean]',
+          'Argument of [mv_zip] must be [string], found value [booleanField] type [boolean]',
+        ]);
```

Comment:
Why are all these mv_zip tests have been added here? It only added the repeat function 🤔 cc @drewdaemon

Reply:
Must not have run `make:tests` manually after updating the `mv_zip` definition. I don't think we need to worry.